src/libraries/Common/src/System/Collections/Generic/EnumerableHelpers.cs

```diff
@@ -8,6 +8,13 @@ namespace System.Collections.Generic
     /// </summary>
     internal static partial class EnumerableHelpers
```

Comment:
Is this still needed after the recent commit?

Reply:
Yes, as `Queue` uses it:

https://github.com/dotnet/runtime/blob/3bd75ba9f1a6a7a8727e53a1a15589ff3a974f30/src/libraries/System.Collections/src/System/Collections/Generic/Queue.cs#L57

Reply:
That should not be needed for `MS_IO_REDIST`. Can the ifdefs in this file be dropped?